src/index.html

```diff
@@ -9,6 +9,24 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>Search bar airbnb</h1>
+
+<div class="wrapper">
+  <div class="search-box search-box-big">
+    <form action="" data-qa="big">
+      <input type="image" src="images/Search.svg" class="icon-search icon-search-big">
```

Comment:
`<input type="image" >` causes page to reload, try to use image as **background** into `<input type="text">`